### PR TITLE
[#2783] Add monitoring data to service

### LINKED ISSUE #2783: Access to EOSC Monitoring Data
We have issued an access token in argo-web-api (devel) to be able to retrieve information about a service using it's resource-id 

How to retrieve the token: 
By visiting: https://dnote.noc.grnet.gr/tDc8BjPrz3VEeNuhKNtUuA 

Please note that you will be asked for a passphrase, use the following: show_me_the_token 

The token will be decrypted - please note that you will not be able to use that link again 

The api endpoint is the following: 
https://api.devel.argo.grnet.gr/api/v3/status/Default/id/{resource-id} 

Some quick examples : 
1.	request to fetch the latest status information about a service endpoint with id: grnet.eosc_monitoring 
2.	curl --request GET 'https://api.devel.argo.grnet.gr/api/v3/status/Default/id/grnet.eosc_monitoring' --header 'Accept: application/json' --header 'x-api-key: s3cr3t-t0k3n' 
2.	request to fetch today's status timeline for the service endpoint with id: grnet.eosc_monitoring 
3.	curl --request GET 'https://api.devel.argo.grnet.gr/api/v3/status/Default/id/grnet.eosc_monitoring?view=details' --header 'Accept: application/json' --header 'x-api-key: s3cr3t-t0k3n' 
3.	request to fetch status timeline for a specific period (Oct 1st till Oct 3rd) for the service endpoint with id: grnet.eosc_monitoring 
4.	curl --request GET 'https://api.devel.argo.grnet.gr/api/v3/status/Default/id/grnet.eosc_monitoring?start_time=2022-10-01T00:00:00Z&end_time=2022-10-03T23:59:59Z' --header 'Accept: application/json' --header 'x-api-key: s3cr3t-t0k3n' 
5.	
Documentation: 
•	More about the specific request: https://argoeu.github.io/argo-monitoring/docs/guides/EOSC-getStatusData 
•	More about argo-web-api: https://argoeu.github.io/argo-web-api 
•	More about argo monitoring in general: https://argoeu.github.io/argo-monitoring

### PR DESCRIPTION
**Pull request by goreck888**
Add details from EOSC monitoring
to the service details page
Closes #2783

https://www.figma.com/file/yRoVOQpeCDEF8rkKYeR7kx/Service-Monitoring?node-id=5%3A686&t=5UUtpDcxspecVY2F-1

https://marketplace-8.docker-fid.grid.cyf-kr.edu.pl/services/eosc.egi-fed.notebook
https://marketplace-8.docker-fid.grid.cyf-kr.edu.pl/services/eosc.athena.uw-map

### CODE REVIEW

File: datasources.json [hunk 1]
```diff
@@ -0,0 +1,3619 @@
+[
```
Comment (goreck888) [inline datasources.json:1]: To delete

**Review by michal-szostak — APPROVED** [review]

**Review by michal-szostak — APPROVED** [review]

**Comment by Marcelinna** [comment]
Open Knowledge Maps
image
image
image

why data in MP is not equal data in ARGO

**Comment by Marcelinna** [comment]
image
image

we take a correct data?

**Comment by Marcelinna** [comment]
Are all services connected?
image
image
image

Why 100 Percent IT Trusted Cloud don't have monitoring board in MP, despite being visible in Argo

**Review by Marcelinna — APPROVED** [review]

**Pull request merged**